packages/itwinui-css/src/tile/tile.scss

```diff
@@ -184,15 +185,19 @@
     display: flex;
     align-items: center;
     margin-top: auto;
-    color: var(--iui-color-foreground-4);
 
     > svg,
     .iui-tile-metadata-icon {
       @include iui-icon-style('m');
-      @include iui-icon-color;
+      fill: currentColor;
```

Comment:
Changed so the icon can change fill between default state & disabled state.  Could also apply the color directly instead of using `currentColor`.